dare.go

```diff
@@ -44,9 +44,9 @@ const (
 
 var (
 	errMissingHeader      = errors.New("dare: incomplete header")
```

Comment:
There is much more to fix, README, spec - we should fix this as soon as we have a stable name... I can add an issue... See #3